fuzz/fuzz.c

```diff
@@ -164,6 +168,11 @@ struct settings {
     const char *output; // path or NULL for tmp file
 };
 
+#ifdef _WIN32
+#define READ_RET int
+#else
+#define READ_RET ssize_t
+#endif
```

Comment:
Could it be in some common header? Also `<io.h>` and `<unistd.h>` could be included from common header.

Also, that's not correct for mingw build. This should be done only for MSVC. And using larger type is fine, makes easier for downstream compatibility.

```suggestion
#ifdef _MSC_VER
typedef long long ssize_t;
#endif
```

Reply:
Win32 defines `ssize_t` as an unsigned type.